pkgs/development/python-modules/async-tkinter-loop/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+, poetry-core
+, pythonRelaxDepsHook
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "async-tkinter-loop";
+  version = "0.8.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "async_tkinter_loop";
+    hash = "sha256-+9AvnYIZMWCbpCEKdbIadyU8zgyUlW/fRYYyDOxAzeg=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    python3Packages.tkinter
```

Comment:
In python-module, we can call `tkinter` directly, like `poetry-core`.

Reply:
This appears to fail on Darwin specifically?

Reply:
I'm not sure... It might relate to #238990

Reply:
I have confirmed that it can build on darwin with python310.
The CI failure is caused by factors other than this PR.

Reply:
Excellent! Thank you :)